packages/build/src/release.ts

```diff
@@ -25,6 +27,16 @@ export default async function release(
   });
 
   const githubRepo = new GithubRepo(config.repo, octokit);
+  const commitTag = await githubRepo.getTagByCommitSha(config.revision);
```

Comment:
What if there are multiple tags on that revision? 🤔 
The proposed process is to use the `rc` tags and then re-tag the latest rc for publishing. However, this will lead to the same commit hash having multiple tags. The `getTagByCommitSha` would then just take the "first one" it `.find()`s. Is this deterministic?

Reply:
No this is just going away. We should just use the evergreen env var, but we probably need another ticket to wire that in.